### ISSUE TITLE
Slider lib in alpaca

### ISSUE THREAD

**Issue by kjugi** [SnowdogApps/magento2-alpaca-components#143]
Lory has some [bug](https://github.com/meandmax/lory/issues/707) which stopping my work on slider.

Tl;DR in the issue: Can't add gutter between slides b/c of bad calc slider width.

I search for some alternatives for Lory and found this: 
+ https://github.com/nolimits4web/Swiper
+ https://github.com/metafizzy/flickity
+ https://github.com/pawelgrzybek/siema
+ https://github.com/jedrzejchalubek/glidejs
+ https://github.com/jssor/slider
+ https://github.com/BE-FE/iSlider
+ https://github.com/lyfeyaj/swipe
+ https://github.com/ruyadorno/simple-slider
+ https://github.com/ganlanyuan/tiny-slider

All of this are not required jQuery, only JavaScript. Don't know about performance ( one of this weight only 1kb) and don't really know what can these sliders. Is it similar to Lory or better/worst

**Comment by Igloczek** [comment]
Apart of that small "bug", which can be considered as a design choice, lory works great and it's extremely light comparing to the competitors, i.e. Swiper is 109KB, while lory only 7.46 KB.

For now, we are going to keep using lory, just add an extra wrapper to add margins inside slides.
